generators/ruby-v2/model/src/object/generateFields.ts

```diff
@@ -12,7 +13,7 @@ export function generateFields({
     context: ModelGeneratorContext;
 }): ruby.AstNode[] {
     return properties.map((prop, index) => {
-        const fieldName = prop.name.name.snakeCase.safeName;
+        const fieldName = toNormalcase(prop.name.name.snakeCase.safeName);
```

Comment:
I'm wary that there are other places we'd need to change this. Like the dynamic snippet generator or pagination generation